reports/descritivo-trt13-2018-2020.Rmd

```diff
@@ -0,0 +1,433 @@
+  geom_point(aes(x = 19.316), size = 4, color = "#111111", shape = 124) +
+  annotate("text", label = "Remuneração media de 2018", x = 3, y = 13, color = "#111111") 
+```
+
+Em 2019 existem funcinários com remuneração acima de **R\$ 300.000,00** e outros com remuneração abaixo de **R\$ 5.000,00**. A média salárial foi de **R\$ 20.568,00**
+
+```{r echo = FALSE, message = FALSE, warning = FALSE, cache = FALSE, paged.print = FALSE}
+dados_filtrados %>% 
+  filter(aid == "trt13", year == 2019) %>% 
+  ggplot(aes(x = income_total/1000, y = month, color = "trt13")) +  
+  theme_minimal() + 
+  scale_color_brewer(palette = "Set2") +
+  geom_point(shape = 124, size = 6, alpha=0.3) + scale_y_continuous(breaks=seq(1, 12, by = 1)) + scale_x_continuous(labels = scales::comma, trans   = "log", breaks=c(5, 10, 19, 50, 150, 350)) + theme(legend.position = "none") + labs(x = "Valor da remuneração (em milhares R$)", y = "Mês", title = "Distribuição das remuneração em 2019") + 
+  geom_point(aes(x = 20.568), size = 4, color = "#111111", shape = 124) +
+  annotate("text", label = "Remuneração media de 2019", x = 3, y = 13, color = "#111111") 
+```
+
+Em 2020 existem funcinários com remuneração acima de **R\$ 100.000,00** e outros com remuneração abaixo de **R\$ 5.000,00**. A média salárial foi de **R\$ 19.458,00**, até então.
+
+```{r echo = FALSE, message = FALSE, warning = FALSE, cache = FALSE, paged.print = FALSE}
+dados_filtrados %>% 
+  filter(aid == "trt13", year == 2020) %>% 
+  ggplot(aes(x = income_total/1000, y = month, color = "trt13")) +  
+  theme_minimal() + 
+  scale_color_brewer(palette = "Set2") +
+  geom_point(shape = 124, size = 6, alpha=0.3) + scale_y_continuous(breaks=seq(1, 12, by = 1)) + scale_x_continuous(labels = scales::comma, trans   = "log", breaks=c(5, 10, 19, 50, 100)) + theme(legend.position = "none") + labs(x = "Valor da remuneração (em milhares R$)", y = "Mês", title = "Distribuição das remuneração em 2020") + 
+  geom_point(aes(x = 19.458), size = 4, color = "#111111", shape = 124) +
+  annotate("text", label = "Remuneração media de 2020", x = 3, y = 13, color = "#111111") 
+```
+
+### Distribuição das remunerações por cargo
+Os Juízes do TRT13 são os que apresentam as maiores remunerações. Logo após estão os Analistas Judiciários. Em terceiro têm-se os Técnicos Judiciários. As categorias Sem Vínculo, Auxiliar Judiciário e e Outros são minoria no TRT13 e apresentam as menores remunerações.
+
+```{r echo = FALSE, message = FALSE, warning = FALSE, cache = FALSE, paged.print = FALSE}
+library(stringr)
+dados_normalized_role <- dados_filtrados %>% 
+  mutate( normalized_role =
+            case_when(
+              str_detect(str_to_lower(role), "analista judiciário") | str_detect(str_to_lower(role), "analista judiciario") | str_detect(str_to_lower(role), "analisa judiciário") ~ "Analista Judiciário",
```

Comment:
Por que o str_detect aqui? == não resolveria? 
Se resolver, existe um operador chamado %in%, você pode fazer str_to_lowerr(role) %in% c("a", "b", "c") e vai retornar true se o valor da esquerda for igual a algum no vetor da direita.

Reply:
É que existem 303 categorias diferentes, só de analista judiciário tem 121.
Por exemplo, Juiz pode ser:
JUIZ CLASSISTA - NÍVEL FUNDAMENTAL JCJC 
JUIZ CLASSISTA DE JCJ 
JUIZ CLASSISTA DO TRT
JUIZ DO TRABALHO SUBSTITUTO
JUIZ DO TRABALHO SUBSTITUTO JSVEN
e mais 10 categorias, todos juízes, sem contar os outros cargos, aí ficaria muito trabalhoso colocar todas as categorias nos arrays, o str_detect facilita bastante, buscando pelos termos base